packages/components-js/tests/e2e/playwright/helpers/sleep.ts

```diff
@@ -0,0 +1,3 @@
+export async function sleep(waitInMs = 0) {
```

Comment:
Can we enforce explicit typing?

Reply:
Need to add ESLint here as well, added it to the follow up.

[https://github.com/orgs/porsche-design-system/projects/14/views/1?pane=issue&itemId=53391745](https://github.com/orgs/porsche-design-system/projects/14/views/1?pane=issue&itemId=53391745)

Reply:
Should be possible to achieve this without eslint.
- https://www.typescriptlang.org/tsconfig#noImplicitAny
- https://www.typescriptlang.org/tsconfig#noImplicitReturns